site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -0,0 +1,169 @@
+<menu popover id=menu1>
+  <li>
+    <button popovertarget=submenu1 popovertargetaction=hover>
+      Sub menu 1
+    </button>
+  </li>
+  <li>
+    <button popovertarget=submenu2 popovertargetaction=hover>
+      Sub menu 2
+    </button>
+  </li>
+</menu>
+<menu class=submenu popover id=submenu1>
+  Sub-menu #1
+</menu>
+<menu class=submenu popover id=submenu2>
+  Sub-menu #2
+</menu>
+<style>
+  button { popover-show-delay: 0.5s; }
+  [popover].submenu { popover-hide-delay: 0.5s; }
+  menu { list-style-type: none; }
+  menu button { border: none; }
+</style>
+```
+
+This example implements a popover menu (`id=menu1`) which has sub-menus (`id=submenu1` and `id=submenu2`) which can be triggered simply by hovering or focusing the main menu's buttons.
+
+# `popover=hint` Behavior
+
+When a hint is shown, it behaves differently from other popover types such as `popover=auto` and `popover=manual` in a few ways:
+
+|               | `popover=auto`            | `popover=hint` | `popover=manual` |
+| ------------- | ------------------------- | -------------- | --------------   |
+| Light dismiss |      Yes                  |     Yes        |      No          |
+| Force-hides:  | Other `auto`s and `hint`s | Other `hint`s  | Nothing          |
+| Nesting:      |     Yes                   | No             | No               |
+
+First, hints should always be light dismissable. They are transient, supplementary information, so they should not require affirmative action to close them. Clicking outside or hitting ESC will close a hint.
+
+Second, when a hint is opened, it should not immediately close other `popover=auto` popovers that are open. For example, if a select picker UI is opened as a `popover=auto`, and the user hovers or focuses a unrelated element to see its hint, that action should not close the select picker. In essence, the hint is *even more transient* than the picker, so it shouldn't take precedence. However, if the user subsequently hovers/focuses a *second* unrelated element and another hint is shown, the first hint should be closed. It would be confusing to have two hints open at a time. So hints should close other hints.
+
+Finally, `popover=auto` popovers support "nesting": the idea that one popover is a descendant of another popover, for example in the case of nested menus. It makes sense in that example for the parent menu to stay open while the child menu opens. The same is not true for hints - it would be confusing for one hint to be nested inside another, and wouldn't make sense. Therefore, hints do not support nesting.
+
+
+# Accessibility
+
+A key distinction between types of hints is whether it may be important for a user to explore or interact with the hint. Consensus on these types of hints/tooltips can help with the resolution of [ARIA#979: Clarify the use of role=tooltip](https://github.com/w3c/aria/issues/979).
+
+- A plain hint contains only elements with a computed role of generic, text and images.
```

Comment:
Given that `aria-describedby` will not properly convey semantics, I don't think that anything other than a text string should be allowed as a "plain hint".

Reply:
Interesting comment! @scottaohara would you agree? I think it sounds pretty limiting to only allow pure text, and not even images.

Reply:
no i don't agree.  that's why there would be the two concepts of a rich hint vs a standard hint.  

per the hello world example above, or even something like the following:

```
<input aria-describedby=foo>
<p id=foo><img src=... alt=warning:> this field cannot be empty</p>
```

the above is perfectly fine to convey as "warning: this field cannot be empty"

as opposed to "warning, image, this field cannot be empty"